### ISSUE TITLE
Issue with blogs when forking

### ISSUE THREAD

**Issue by ColeFeely6** [alshedivat/al-folio#1195]
Hi! Thank you for the awesome template, I have been working with it and it's quite wonderful. Everything has been working great but when I have started to post blogs to the site, I have an issue where when you click on the tags to get posts related to that site, it gives you a page not found.

![image](https://user-images.githubusercontent.com/65377055/219883949-2b36afe0-8cc9-4d0d-ad82-4928d9b0d91b.png)

[DEEPER IN DEPTH DESCRIPTION](https://stackoverflow.com/questions/75526355/tags-and-category-list-pages-for-blog-not-loading-on-jekyll-site)

I don't believe that I have changed anything that could mess this up, you can check out my live version at https://colefeely6.github.io/ to see what I am talking about. The blog posts are on the site, like you can go to the default posts like  https://colefeely6.github.io/blog/2015/code/ or one I posted https://colefeely6.github.io/blog/2023/wisconsin-method/, the posts exists, but if you click on the tag from the blog landing page or from the blog post, it gives a page not found. Do you have any idea why this may be happening? I would like to have a blog where I can have my own tags as well. 

Again thank you so much, just been having trouble with this bug.

**Comment by ColeFeely6** [comment]
The issue seems to be that the _layout/archive-tag.html, _layout/archive-category.html and _layout/archive-year.html are not working but whichever file that loads the actual blog post is working, either _post or _page. So the issue must be with whatever loads these because the code is exactly the same as the original site.

**Comment by ColeFeely6** [comment]
Note for self: https://jekyllrb.com/docs/posts/

**Comment by ColeFeely6** [comment]
A temproary fix I found was replacing the code going through the tags and categories in /blog/index.html with just:

  {% for tag in site.tags %}
  <h3>{{ tag[0] }}</h3>
  <ul>
    {% for post in tag[1] %}
      <li><a href="{{ post.url }}">{{ post.title }}</a></li>
    {% endfor %}
  </ul>
{% endfor %}

This was found in the Jekyll documentation. If I find the time and I really care about fixing this issue, I'll go through the original code.

**Comment by george-gca** [comment]
Was it working before you started implementing your changes? The easiest way to debug this is to reapply all the changes one by one, preferably in small changes. I tried to look at your code now, but since then a lot of changes were already made.

**Comment by tostenzel** [comment]
I have the same issue, but only remote -- not locally on my machine.

**Comment by george-gca** [comment]
Are you deploying manually, like suggested in the [README](https://github.com/alshedivat/al-folio#deployment) (in the `Deployment to another hosting server (non GitHub Pages)` section)?

**Comment by george-gca** [comment]
TBH I have no idea. I always used Github sites. Maybe one of the repo owners can answer this.

But now I just realized one thing. In your `_config.yml` your url looks like this: `url: https://tobiasstenzel.com`
If you copy the url from there and paste it in your browser, it doesn't work. Why? Because in your case it is missing the `www.`. Maybe if you add `www.` to your `url` it works?

**Comment by tostenzel** [comment]
Good catch! :-) Did not help though :/

What's the best option to ask the owners directly? I think there is no direct messaging functionality on github, right?

**Comment by tostenzel** [comment]
@alshedivat

**Comment by ColeFeely6** [comment]
@tostenzel how were you able to get your blog posts displaying like this? Mine is having issues, I had to make a manual workaround:

Yours: 

<img width="1242" alt="Screenshot 2023-03-21 at 4 13 36 PM" src="https://user-images.githubusercontent.com/65377055/226729964-4893e42c-1513-4827-8e79-80c9a7dc2b78.png">

Mine: 

https://colefeely6.github.io/blog/

<img width="1260" alt="Screenshot 2023-03-21 at 4 14 22 PM" src="https://user-images.githubusercontent.com/65377055/226730132-4f30895c-0b20-4353-be9f-eff7b63f9cdc.png">

**Comment by tostenzel** [comment]
To what do you refer specifically? The original template does also look different than yours, see https://alshedivat.github.io/al-folio/blog/

**Comment by tostenzel** [comment]
For instance, do you specify the hashtags in the head of the markdown files of the blogposts?

**Comment by tostenzel** [comment]
I guess this is the change I did in the distill template. However, this might also be present in the post template that you are using;
https://github.com/alshedivat/al-folio/compare/master...tostenzel:tostenzel.github.io:master#diff-dd8ca147cf4c0469aa879c10a03b9c9e1183486699aadcb0d14f52f8ac398cc4

Also note that the tags do not work remotely for my site... :(

**Comment by tostenzel** [comment]
I think I will re-do everything from the start. Perhaps that's also a good idea for you @ColeFeely6 . Shouldn't be such a hustle if we save our blog files.

**Comment by george-gca** [comment]
This would also be my recommendation for you guys. Start from the latest version of the template, and just re-do everything. Maybe doesn't even need to be too manual. You could create a branch, fetch the latest version of the template, and reapply commit by commit, testing between each commit (both locally and online).

But from the start @tostenzel, check if the template works in you custom domain (I mean, the tag part). If even then it didn't work, try to do the copy/paste version stated in the README, and check if that works.

**Comment by tostenzel** [comment]
Good advice, thanks!

**Comment by KunalA18** [comment]
I am also facing a similar issue.
Blogs are shown locally but not after being deployed. 

(after inspecting, found out that the "ul" list is not getting deployed(not visible on inspecting atleast))

My repository: [https://github.com/KunalA18/KunalA18.github.io/tree/gh-pages](https://github.com/KunalA18/KunalA18.github.io/tree/gh-pages)
My website: [https://kunala18.github.io/blog/](https://kunala18.github.io/blog/)

Can someone help? Thanks

**Comment by george-gca** [comment]
It is hard to debug someone else's code, especially after so many changes. I recommend you follow [these practices](https://github.com/alshedivat/al-folio/issues/1195#issuecomment-1478635920). This way it is easier to find out when it breaks and then look for a solution.

**Comment by MarkMorow** [comment]
I had a similar issue and I'm not sure if this is what you all are seeing as well but I figured I'd share it. I noticed when I clicked on any of the 'tags' that you configure in the config.yml file I would get a file not found. I only had 1 post so that's what I expected to see for that tag. The default sample post type in the header is 'category'. When I changed that to be 'tags:' in the header of the post it then worked correctly for me.
